packages/types/src/integrations.ts

```diff
@@ -71,6 +71,7 @@ export type CommandResponse = {
     channel_id: SVGAnimatedString;
     icon_url: string;
     type: string;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
     props: Record<string, any>;
```

Comment:
we dont need to line disable eslint here since `any` is a warning not error in our config